module/account/src/Application/Form/Model/GenerateUserTokenModel.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Account\Application\Form\Model;
+
+use Ergonode\Account\Application\Validator\Constraints\AvailableHostConstraint;
+use Symfony\Component\Validator\Constraints as Assert;
+
+class GenerateUserTokenModel
+{
+    /**
+     * @Assert\NotBlank(message="Email is required")
+     * @Assert\Email(mode="strict")
+     */
+    public ?string $email;
+
+    /**
+     * @Assert\NotBlank()
+     * @Assert\Url()
+     *
+     * @AvailableHostConstraint(site="path")
+     */
+    public ?string $path;
```

Comment:
```suggestion
    public ?string $url;
```
path is just a part without host name etc